pycares/_cfficore/__init__.py

```diff
@@ -0,0 +1,667 @@
+#!/usr/bin/env python
+
+from _pycares_cffi import ffi as _ffi, lib as _lib
+import _cffi_backend
```

Comment:
Note that some bundler tools that try to find all modules used by a project, like PyInstaller, will miss _cffi_backend in the out-of-line mode because your program contains no explicit import cffi or import _cffi_backend. You need to add _cffi_backend explicitly (as a “hidden import” in PyInstaller, but it can also be done more generally by adding the line import _cffi_backend in your main program).

https://cffi.readthedocs.io/en/latest/cdef.html